draft-ietf-webtrans-http3.md

```diff
@@ -195,55 +196,54 @@ with an H3_ID_ERROR error code.
 
 ## Unidirectional streams
 
-Once established, both endpoints can open unidirectional streams.  The HTTP/3
-unidirectional stream type SHALL be 0x54.  The body of the stream SHALL be the stream
-type, followed by the session ID, encoded as a variable-length integer, followed
-by the user-specified stream data ({{fig-unidi}}).
+Once a WebTransport session is established, either endpoint can open a
+unidirectional stream as described {{Section 6.3 of HTTP3}}.  This is indicated
+by a stream type of 0x54, followed by the session ID, encoded as a
+variable-length integer. The remainder of the stream carries unframed,
+user-specified, data.
 
 ~~~~~~~~~~ drawing
-  0                   1                   2                   3
-  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                           0x54 (i)                          ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                        Session ID (i)                       ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                         Stream Body                         ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+WebTransport Unidirectional Stream Header {
+  Stream Type (i) = 0x54,
+  Session ID (i),
+}
 ~~~~~~~~~~
-{: #fig-unidi title="Unidirectional WebTransport stream format"}
+{: #fig-unidi title="Unidirectional WebTransport Stream Header"}
 
 ## Bidirectional Streams
 
-WebTransport endpoints can initiate bidirectional streams by opening an HTTP/3
-bidirectional stream and sending an HTTP/3 frame with type
-`WEBTRANSPORT_STREAM` (type=0x41).  The format of the frame SHALL be the frame
-type, followed by the session ID, encoded as a variable-length integer,
-followed by the user-specified stream data ({{fig-bidi-client}}).  The frame
-SHALL last until the end of the stream.
+HTTP/3 reserves client-initiated bidirectional streams for request and
+responses, and does not use server-initiated bidirectional streams; see
+{{Section 6.1 of HTTP3}}.
+
+WebTransport extends HTTP/3 in order to permit endpoints to explicitly declare
+the purpose of bidirectional streams. This is done by sending a stream type,
```

Comment:
It took me a second to grok this, so I'd suggest clarifying the fact that WebTransport is now using HTTP/3's unidirectional stream types for bidirectional streams. Luckily the IANA registry is just called "HTTP/3 Stream Types" so we're good there but making this clearer would help.